src/WpBookingsFrontUi.php

```diff
@@ -50,192 +51,169 @@ public function __construct(
+                    'wizardSessionStepTemplate'      => $c->get('eddbk_front_session_step_template'),
+                    'wizardConfirmationStepTemplate' => $c->get('eddbk_front_confirmation_step_template'),
+                ]);
+            },
+
+            'eddbk_wizard_assets_urls_map' => function (ContainerInterface $c) {
+                $containerFactory = $this->_getContainerFactory();
+                $assetsUrlsMap = require_once $c->get('bookings_front_ui/assets_urls_map_path');
+
+                return $containerFactory->make([
+                    ContainerFactoryInterface::K_DATA => $assetsUrlsMap,
+                ]);
+            },
+
+            'eddbk_wizard_assets' => function (ContainerInterface $c) {
+                return $c->get('bookings_front_ui/assets');
+            },
+
+            /*
+             * Handles assets enqueuing for application.
+             *
+             * @since [*next-version*]
+             */
+            'eddbk_wizard_enqueue_assets_handler' => function (ContainerInterface $c) {
+                return new AssetsEnqueueHandler($c->get('eddbk_wizard_assets_urls_map'), $c->get('eddbk_wizard_assets'));
+            },
+
+            /*
+             * Handles state output for client application.
+             *
+             * @since [*next-version*]
+             */
+            'eddbk_wizard_enqueue_app_state_handler' => function (ContainerInterface $c) {
+                return new StateEnqueueHandler(
+                    $c->get('bookings_front_ui/application_selector'),
+                    $c->get('bookings_front_ui/api_endpoint_urls'),
+                    $c->get('bookings_front_ui/booking_data_map'),
+                    $c->get('bookings_front_ui/initial_booking_transition'),
+                    $c->get('bookings_front_ui/formats/datetime')
+                );
+            },
+
+            /*
+             * Handles for outputting main component of application.
+             *
+             * @since [*next-version*]
+             */
+            'eddbk_wizard_main_component_handler' => function (ContainerInterface $c) {
+                return new MainComponentHandler(
+                    $c->get('eddbk_front_app_holder_template')
+                );
+            },
+        ]);
     }
 
     /**
-     * Get content of template file.
-     * 
+     * Create template instance from file.
+     *
      * @since [*next-version*]
-     * 
-     * @param string $templateFile Template file for reading.
-     * 
-     * @return string Content of given template file.
+     *
+     * @param TemplateFactoryInterface $templateFactory Template factory for creating template.
+     * @param string                   $templateFile    Template file for reading.
+     *
+     * @return TemplateInterface The created template.
      */
-    protected function _openTemplate($templateFile)
+    protected function _makeTemplate($templateFactory, $templateFile)
```

Comment:
The factory is an implementation detail, and therefore `_makeTemplate()` should not accept it, but should be internally aware of it.